networkd/network_server.ml

```diff
@@ -15,9 +15,9 @@
 open Network_utils
 open Network_interface
 
-open Fun
-open Xstringext
-open Listext
+(*open Fun*)
```

Comment:
Ah, I think I commented it out to see if it was using any of the other funky things declared in Fun, but seemingly not :-)